.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -9,9 +9,11 @@ assignees: ''
 
 - [ ] Check [Nightlies](http://cdash.seqan.de/index.php?project=SeqAn3) for critical build failures
 - [ ] Check Changelog.md for completeness
-- [ ] Prepare `seqan-[VERSION]-with-submodules.tar.gz` 
+- [ ] Prepare `seqan-[VERSION]-with-submodules.tar.gz`
 - [ ] Tag release on Github and attach `seqan-[VERSION]-with-submodules.tar.gz` to the release
 - [ ] Bump succeeding version number
+- [ ] update the index from cppreferences.com so that up-to-date documentation links are generated.
```

Comment:
This includes making changes to SeqAn so it needs to happen before tagging the release. In general tagging the release should happen as late as possible to avoid unfixable errors.

Reply:
The other one too?